futures-util/src/try_stream/try_filter_map.rs

```diff
@@ -0,0 +1,88 @@
+use core::marker::Unpin;
+use core::mem::PinMut;
+use futures_core::future::{TryFuture};
+use futures_core::stream::{Stream, TryStream};
+use futures_core::task::{self, Poll};
+
+/// A combinator that attempts to filter the results of a stream
+/// and simultaneously map them to a different type.
+///
+/// This structure is returned by the `Stream::try_filter_map` method.
```

Comment:
Nit: can you make this include a doc link? like [this one](https://github.com/rust-lang-nursery/futures-rs/blob/master/futures-util/src/try_stream/err_into.rs#L6)